packages/icons/src/index.tsx

```diff
@@ -2,6 +2,31 @@
 
 import * as React from 'react'
 
+export const ArrowBack = React.forwardRef(function ArrowBack(
+  props: React.SVGProps<SVGSVGElement>,
+  svgRef?: React.Ref<SVGSVGElement>
+) {
+  return (
+    <svg
+      width={24}
+      height={24}
+      viewBox='0 0 24 24'
+      fill='currentColor'
+      ref={svgRef}
+      {...props}
+    >
+      <path
+        d='M20 12H4m6 6l-6-6 6-6'
+        fill='none'
+        stroke='currentColor'
+        strokeWidth={2}
+        strokeLinecap='round'
+        strokeLinejoin='round'
+      />
+    </svg>
+  )
+})
```

Comment:
It's not good idea to convert svg-s to React components manually, especially when we have duplicate in svg format.
It would be better to use utility for it. But again, **we can write it as one of improvements for ui lib, and refactor in future.** Now is not critical.

Reply:
That's done not manually, we have a converter script (which uses svgo) in this repo for this purpose.
See this: [Adding a new icon](https://github.com/lidofinance/ui/blob/81b86d09776aea7e1a9ab6d6be6eb3f2e5408b40/packages/icons/README.md#adding-a-new-icon)

Reply:
Anyway, u need to run command manually. I mean to use as https://www.npmjs.com/package/@svgr/webpack.

Clear, one source of true. 

```
import Star from './star.svg'

const App = () => (
  <div>
    <Star />
  </div>
)
```